Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -196,9 +271,21 @@ module.exports = class DoubleClickToEdit {
 		if (!message)
 			return;
 
-		if (message.author.id === this.CurrentUserStore.getCurrentUser().id)
+		//Now that we know it has to be a message, we check to copy the text before anything else changes.
+		const copyKeyHeld = this.checkForModifier(this.copyEnabled, this.copyModifier, e);
+		if (this.copyEnabled && copyKeyHeld) {
```

Comment:
I missed this yesterday, but `this.copyEnabled` is already caught in `this.checkForModifier` as the first argument.

So then it'd only be `if (copyKeyHeld)`.